numba_dpex/_patches.py

```diff
@@ -265,6 +325,7 @@ def _empty_nd_impl(context, builder, arrtype, shapes):
         fnop.get_call_type(context.typing_context, sig.args, {})
         eqfn = context.get_function(fnop, sig)
         meminfo = eqfn(builder, args)
+        pyapi.decref(queue_ptr)
```

Comment:
Maybe move to end of the function.

Reply:
Moved to the end but still gives segfault on `blackscholes_njit.py`